tests/src/blockchainProxy.unit.test.ts

```diff
@@ -0,0 +1,44 @@
+import {KinAccount} from "../../scripts/src/kinAccount";
+import {Environment} from "../../scripts/src/environment";
+import {KeyPair, KinClient} from "../../scripts/src";
+import * as KinSdk from "@kinecosystem/kin-sdk";
+
+const keyPair = KeyPair.generate();
+let client: KinClient;
+let sender: KinAccount;
+
+describe("Blockchain Proxy", async () => {
+
+	const Proxy = new Environment({
+			name: "Test",
+			url: "https://haproxy.kininfrastructure.com",
```

Comment:
should be a local test (don't commit it)

Reply:
Can you please explain why not?

Reply:
cause proxy url is temporary and will be removed, and because those tests are similar to other integ tests and seems redundant to me. 
besides that, now, for instance, this url shouldn't redirect without the special header,
so to sum it up, this test should be local by replacing network url at the rest of integ test, and doing this online while verifying the correct redirect strategy of this url (redirect only with header? always?) it can be tested in the shell using `curl -v`. 
I may add a test verifying redirects works as expected by simulating it using nock, for instance, https://stackoverflow.com/a/35467859/3903847